src/note/commitment.rs

```diff
@@ -78,9 +84,15 @@ impl ExtractedNoteCommitment {
         self.0.to_repr()
     }
 
-    pub(crate) fn inner(&self) -> jubjub::Base {
+    /// Returns the inner [`jubjub::Base`] value
+    pub fn inner(&self) -> jubjub::Base {
         self.0
     }
+
+    /// Returns the inner [`jubjub::Base`] value
+    pub fn from_base_unchecked(base: jubjub::Base) -> Self {
```

Comment:
Documentation is wrong.

Can you provide an example of where you want to use this, that is not satisfied by `ExtractedNoteCommitment::from_bytes`?

Reply:
Our use case is satisfied by `from_bytes` if we replace `NoteCommitment` and remove the `extract_p()` defined in `zebra-chain` too, which we should do. If we were to only use `ExtractedNoteCommitment` without replacing our `NoteCommitment` type, this would help avoid encoding/decoding the base at the expense of potential misuse. I'll take this out.

Reply:
Method was removed.